pkg/build/build.go

```diff
@@ -29,6 +30,7 @@ type Context struct {
 	WorkDir            string
 	TarballPath        string
 	UseProot           bool
+	SourceDataEpoch    time.Time
```

Comment:
Should this not be `SourceDateEpoch`?

Reply:
It should, good catch :+1: 

note to self: I need to sleep :facepalm: